projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/job_data.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+import pathlib
+
+import ipykernel
+from vdk.internal.builtin_plugins.config.job_config import JobConfig
+
+try:
+    import hybridcontents
+except ImportError:
+    hybridcontents = None
+
+
+class JobData:
+    """
+    class responsible for retrieving data for the job
```

Comment:
nit: The class name may impress an object like model or DTO. Yet the docs suggest it does something, it performs actions in favour of purely storing data. The suggestion is to have it named in a way, that suggests the action